tests/Resources/ti.network.httpclient.test.js

```diff
@@ -735,4 +735,26 @@ describe('Titanium.Network.HTTPClient', function () {
 		xhr.open('GET', 'https://www.google .com/'); // URL with space
 		xhr.send();
 	});
+
+	it.ios('#timeoutForResource', function (finish) {
+		const xhr = Ti.Network.createHTTPClient({
+			timeout: 6e4,
+			timeoutForResource: 50
```

Comment:
50 ms is very short, and unlikely it'll ever pass, but maybe just in case we make it like 5ms?